multimedia/Sunshine/Portfile

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem               1.0
+PortGroup                cmake 1.1
+PortGroup                github 1.0
+PortGroup                boost 1.0
+
+# bump revision when changes are made to this file
+revision                 1
+
+github.setup             sunshinestream sunshine master
+name                     Sunshine
+version                  0.14.0
+categories               multimedia emulators games
+platforms                darwin
+license                  GPL-3
+maintainers              {@SunshineStream sunshinestream}
+description              Sunshine is a Gamestream host for Moonlight.
+long_description         {*}${description}
+homepage                 https://sunshinestream.github.io
+master_sites             https://github.com/sunshinestream/sunshine/releases
+
+fetch.type               git
+post-fetch {
+    system -W ${worksrcpath} "${git.cmd} submodule update --init --recursive"
+}
+
+depends_lib              port:avahi \
+                         port:ffmpeg \
+                         port:libopus
+
+boost.version            1.76
```

Comment:
You appear to not like being explicit... In my opinion it's better to explicitly state what you want, so when someone changes the defaults, it still works...

Reply:
I am not sure what you mean with "to not like being explicit"; we have defaults set by PortGroups or `macports-base` for a reason, namely such that every individual port doesn't need to repeat that in its Portfile.

Ideally we want ports to use the specific version of boost (or some other things like Python, Perl, ...) that MacPorts considers the default and only override that when a package doesn't build with it. Typically, a change of the default versions will be announced on the "-devel" mailinglist giving maintainers time to check whether "their" ports build with that new default, and, if not, to specify a compatible version.

Having said that, if you want to specify `boost.version` that's fine, I just pointed out that it is not required since you're repeating the default.